packages/cdk-graph-plugin-diagram/README.md

```diff
@@ -0,0 +1,335 @@
+## Diagram Plugin - Cdk Graph
+`@aws-prototyping-skd/cdk-graph-plugin-diagram`
+
+![experimental](https://img.shields.io/badge/stability-experimental-orange.svg)
+![alpha](https://img.shields.io/badge/version-alpha-red.svg) \
+[![API Documentation](https://img.shields.io/badge/view-API_Documentation-blue.svg)](https://aws.github.io/aws-prototyping-sdk/typescript/cdk-graph-plugin-diagram/index.html)
+[![Source Code](https://img.shields.io/badge/view-Source_Code-blue.svg)](https://github.com/aws/aws-prototyping-sdk/tree/mainline/packages/cdk-graph-plugin-diagram)
+
+This plugins generates diagrams utilizing the [cdk-graph](https://aws.github.io/aws-prototyping-sdk/typescript/cdk-graph/index.html) framework.
+
+> More comprehensive documentation to come as this package stabilizes.
+
+> **Disclaimer:** This is the first **cdk graph** plugin, it is highly *experimental*, and subject to major refactors as we gain feedback from the community.
+
+| | |
+| --- | --- |
+| <img src="docs/examples/default.png" width="300" /> | <img src="docs/examples/dark.png" width="300" /> |
+
+
+### Quick Start
+```ts
+// bin/app.ts
+(async () => {
```

Comment:
We should add some more docs regarding the async wrapper and that people will need to ensure this surrounds their apps.